docs/api/browser-window.md

```diff
@@ -1397,12 +1397,17 @@ can still bring up the menu bar by pressing the single `Alt` key.
 
 Returns `Boolean` - Whether the menu bar is visible.
 
-#### `win.setVisibleOnAllWorkspaces(visible)`
+#### `win.setVisibleOnAllWorkspaces(visible[, visibleOnFullScreen])`
 
 * `visible` Boolean
 
 Sets whether the window should be visible on all workspaces.
 
+* `visibleOnFullScreen` Boolean (optional)
```

Comment:
Final thing I think from me, this should be tagged as `_macOS_` as it is only supported on that platform

Reply:
This also needs to be under the first bullet point, must be a consecutive list of params

Reply:
adjusted!